src/pages/home/platform-carousel/_mobile-platform-carousel.tsx

```diff
@@ -103,50 +103,64 @@ const MobilePlatformCarousel = () => {
-                                title={title}
-                                icon={icon}
-                                description={description}
-                                learn_more_link={learn_more_link}
-                            />
-                        </Flex>
-                        <Flex
-                            ai="flex-start"
-                            jc="center"
-                            fw="wrap"
-                            width="unset"
-                            tabletL={{ m: '3.2rem 3.8rem' }}
-                            mobileL={{ m: '32px 0 40px' }}
-                        >
-                            {download_links.map((link) => (
-                                <DownloadLink
-                                    key={link.type}
-                                    external="true"
-                                    type={link?.link_type}
-                                    to={link?.url}
-                                    target="_blank"
-                                    rel="noopener noreferrer"
-                                >
-                                    <img src={getOSIcon(link.type)} alt={link.type} />
-                                </DownloadLink>
-                            ))}
-                        </Flex>
-                    </CarouselItemWrapper>
-                )
-            })}
+            {carouselData().map(
+                ({ image_key, title, icon, description, learn_more_link, download_links }) => {
+                    return (
+                        <CarouselItemWrapper key={image_key}>
+                            <Flex tabletL={{ mb: '56px' }}>
+                                <MobileImage
+                                    data={images[image_key]}
+                                    alt={image_key}
+                                    height={'55vw'}
+                                />
+                            </Flex>
+                            <Flex>
+                                <PlatformDetails
+                                    title={title}
+                                    icon={icon}
+                                    description={description}
+                                    learn_more_link={learn_more_link}
+                                />
+                            </Flex>
+                            <Flex
+                                ai="flex-start"
+                                jc="center"
+                                fw="wrap"
+                                width="unset"
+                                tabletL={{ m: '3.2rem 3.8rem' }}
+                                mobileL={{ m: '32px 0 40px' }}
+                            >
+                                {download_links.map((link) => (
+                                    <DownloadLink
+                                        key={link.type}
+                                        external="true"
```

Comment:
It doesn't look good to pass string to boolean type.
it would be better to only pass `external` instead of passing `external="true"`